vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -31,11 +31,45 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - curl -sSL \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" | sed \"s:/usr/bin:/usr/local/bin:g\" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
       - usermod -aG docker metal3
       - systemctl enable --now docker keepalived kubelet
+      - systemctl link /etc/systemd/system/multi-user.target.wants/monitor.keepalived.service
+      - systemctl enable --now monitor.keepalived.service
     postKubeadmCommands:
       - mkdir -p /home/metal3/.kube
       - cp /etc/kubernetes/admin.conf /home/metal3/.kube/config
       - chown metal3:metal3 /home/metal3/.kube/config
     files:
+      - path: /usr/local/bin/monitor.keepalived.sh
+        owner: root:root
+        permissions: '0755'
+        content: |
+            #!/bin/bash
+            while :; do
+              curl -sk https://127.0.0.1:6443/healthz 1>&2 > /dev/null
+              isOk=\$?
+              isActive=\$(systemctl show -p ActiveState keepalived.service | cut -d'=' -f2)
+              if [ \$isOk == \"0\" ] &&  [ \$isActive != \"active\" ]; then
+                logger 'API server is healthy, however keepalvied is not running, starting keepalived'
```

Comment:
typo

Reply:
Fixed. This PR merged too early. 
Created another PR. https://github.com/metal3-io/metal3-dev-env/pull/317